kml/serdes_gpx.cpp

```diff
@@ -358,6 +368,83 @@ std::string GpxParser::BuildDescription() const
   return m_description + "\n\n" + m_comment;
 }
 
+namespace
+{
+
+std::optional<std::string> GetDefaultLanguage(LocalizableString const & lstr)
+{
+  auto const firstLang = lstr.begin();
+  if (firstLang != lstr.end())
+    return {firstLang->second};
+  else
+    return {};
```

Comment:
nit: else is redundant.
```suggestion
  return {};
```

Reply:
Fixed